### DISCUSSION TITLE
Welcome to nuxt-three Discussions!

### DISCUSSION THREAD

**Discussion by clementroche** [clementroche/nuxt-three#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ronca85** [comment]
Hello,

My name is Marko. I'm relatively new to nuxt. When I was starting a new project a while ago I was considering this starter but I didn't use it in the end. It seemed to include too many things that are new to me which I wouldn't be able to understand and maintain as the project moves on.

What plugin would you recommend for use with nuxt for reveal-on-scroll effects? I tried using gsap scrollTrigger but it makes such a mess that I'm thinking about using something else.

Thank you.

**Reply by clementroche** [reply]
This starter is not maintained anymore, a lot of things need to be rebuild, maybe one day.

About reveal on scroll you can create your own directive using Intersection Observer API. Otherwise you can use https://github.com/Akryum/vue-observe-visibility
